docs/guide/content_rendering.md

```diff
@@ -584,6 +584,41 @@ To avoid such situations, you can check if the Location is virtual using the `lo
     </div>
     ```
 
+    ### Landing Page template
+    Once published, Landing Page will be displayed using the template according to the `content_view` setting, see [View Matchers](#view-matchers). If you want to see the Landing Page displayed using a particular template in the edit mode of Landing Page Editor before publish, you need to configure additional settings as follow in `ezplatform.yml` configuration file.
+    
+    ``` yml
+    ezstudioui:
+        system:
+            # Defines the scope: a valid SiteAccess, SiteAccess group or even "global"
+            front_siteaccess:
+                studio_template: AppBundle:studio:template.html.twig
+    ```
+    
+    This is an example of a minimal template file:
+    
+    ``` html
+    {% extends base_template() %}
+    {% block content %}
+    
+        <!-- Custom template header code -->
+        
+        <!-- This part is required! -->
+        {% if content is defined %}
+            {{ ez_render_field(content, 'page') }}
+        {% else %}
+            <div data-area="static" style="min-height:300px;"></div>
+        {% endif %}
+        <!-- End required part -->
+        
+        <!-- Rest of the custom template code -->
+        
+    {% endblock %}
+    ```
+    
+    !!! caution
+    Custom template always need to extend `base_template()`. Morevoer, you have to check whether the `content` variable is defined to correctly display previously published Landing Page. Otherwise, you need to display `<div data-area="static"></div>` which is the place where you can put the new blocks. 
```

Comment:
`always needs to extend`
`to correctly display a previously published Landing Page`

Also, the body of the caution box must be double-indented (8 spaces), or it doesn't render correctly.